tests/swagger_generation/registries/hidden_api.py

```diff
@@ -0,0 +1,621 @@
+            "get": {
+                "operationId": "list_foos",
+                "responses": {
+                    "200": {
+                        "description": "Foo",
+                        "schema": {
+                            "type": "array",
+                            "items": {"$ref": "#/definitions/Foo"},
+                        },
+                    },
+                    "default": {
+                        "description": "Error",
+                        "schema": {"$ref": "#/definitions/Error"},
+                    },
+                },
+                "security": [],
+            }
+        },
+        "/foos": {
+            "get": {
+                "operationId": "nested_foos",
+                "responses": {
+                    "200": {
+                        "description": "NestedFoosSchema",
+                        "schema": {"$ref": "#/definitions/NestedFoosSchema"},
+                    },
+                    "default": {
+                        "description": "Error",
+                        "schema": {"$ref": "#/definitions/Error"},
+                    },
+                },
+                "parameters": [
+                    {
+                        "name": "name",
+                        "in": "query",
+                        "required": False,
+                        "type": "string",
+                    },
+                    {
+                        "name": "other",
+                        "in": "query",
+                        "required": False,
+                        "type": "string",
+                    },
+                ],
+                "security": [],
+            }
+        },
+        "/tagged_foos": {
+            "get": {
+                "tags": ["bar", "baz"],
+                "operationId": "tagged_foos",
+                "responses": {
+                    "default": {
+                        "description": "Error",
+                        "schema": {"$ref": "#/definitions/Error"},
+                    }
+                },
+            }
+        },
+    },
+    "definitions": {
+        "Foo": {
+            "type": "object",
+            "title": "Foo",
+            "properties": {"uid": {"type": "string"}, "name": {"type": "string"}},
+        },
+        "FooUpdateSchema": {
+            "type": "object",
+            "title": "FooUpdateSchema",
+            "properties": {"name": {"type": "string"}},
+        },
+        "NestedFoosSchema": {
+            "type": "object",
+            "title": "NestedFoosSchema",
+            "properties": {
+                "data": {"type": "array", "items": {"$ref": "#/definitions/Foo"}}
+            },
+        },
+        "Error": {
+            "type": "object",
+            "title": "Error",
+            "properties": {"message": {"type": "string"}, "errors": {"type": "object"}},
+            "required": ["message"],
+        },
+    },
+}
+
+
+EXPECTED_HIDDEN_SWAGGER_V3 = expected_swagger = {
```

Comment:
Similar to above comment, I would consider giving this one the "normal" base name of `EXPECTED_SWAGGER_V3` and give the other a "specialized name".  Particularly since this is along side `EXPECTED_SWAGGER_V2`.